test/functional/profile-toolbars.js

```diff
@@ -0,0 +1,20 @@
+describe('profile-toolbars', () => {
+  before(() => {
+    // Set viewport size
+    browser.setViewportSize({
+      width: 1400,
+      height: 600,
+    });
+
+    browser.pause(1000);
+    browser.url('ecl-profile-topbars.html');
+  });
+
+  // Normal state
+  it('should match the reference screenshot', () => {
+    const screenshots = browser.checkDocument({
+      name: 'profile-toolbars',
+    });
+    expect(screenshots).to.matchReference();
+  });
```

Comment:
Please add accessibility tests too, as in other test files:

    it('should be accessible', () => {
      const a11yReport = browser.runAxeCore('ecl-profile-topbar').value;
      expect(a11yReport).to.be.accessible;
    });